ksqldb-serde/src/test/java/io/confluent/ksql/serde/json/KsqlJsonSerializerTest.java

```diff
@@ -233,7 +236,7 @@ public void shouldHandleNestedStruct() throws IOException {
   }
 
   @Test
-  public void shouldSerializeBoolean() {
+  public void shouldSerializeBoolean() throws IOException, RestClientException {
```

Comment:
nit: I'm with @big-andy-coates on preferring tests simply have `throws Exception` so we don't need to worry about keeping lists of exceptions up to date. I don't think seeing the specific exceptions is insightful either.